### PR TITLE
LTP: Introduce var to enable reboot after installation

### PR DESCRIPTION
**Pull request by richiejp**
It appears the LTP installation module on SLE12-SP4 x86_64 publishes a
corrupted disk image which can not be booted (or even mounted). This
reintroduces a reboot to the LTP installation, but only if LTP_INSTALL_REBOOT
is set to one. So we can see if it is really the LTP installation corrupting
the disk image or the publishing process.

@pevik

### CODE REVIEW

**Comment by pevik** [comment]
@richiejp LGTM, let me just test it.

**Comment by pevik** [comment]
@richiejp Both jobs I'm testing with `LTP_INSTALL_REBOOT=1` are looping for more than hour in isotovideo status
http://quasar.suse.cz/tests/473#live
http://quasar.suse.cz/tests/474#live

```
[2018-06-20T14:09:15.0895 CEST] [debug] Routing to a callback
[2018-06-20T14:09:15.0896 CEST] [debug] 200 OK (0.000623s, 1605.136/s)
[2018-06-20T14:09:16.0390 CEST] [debug] GET "/8qGeCQHN_xFSDLwn/isotovideo/status" (f168b1ba)
[2018-06-20T14:09:16.0390 CEST] [debug] Routing to a callback
[2018-06-20T14:09:16.0390 CEST] [debug] 200 OK (0.000521s, 1919.386/s)
...
```

File: tests/kernel/install_ltp.pm [hunk 1]
```diff
@@ -281,7 +281,7 @@ sub run {
     upload_runtest_files('${LTPROOT:-/opt/ltp}/runtest', $tag);
 
     select_console('root-console');
-    #type_string "reboot\n";
+    type_string "reboot\n" if get_var('LTP_INSTALL_REBOOT');
```
Comment (rwx788) [inline tests/kernel/install_ltp.pm]: Have you considered using power_action? I know it's not directly related to the change you propose, but this won't work on zVM, for example, or ipmi. And power_action has all the tweaks for different setups.

Comment (richiejp) [reply tests/kernel/install_ltp.pm]: ok, thanks

**Comment by richiejp** [comment]
@pevik I have no idea what that means

File: tests/kernel/install_ltp.pm (hunk 1 above)
Comment (rwx788) [reply tests/kernel/install_ltp.pm]: Just give it a try, if it doesn't work straight away, let's merge this PR and fix it later. Cheers!

**Pull request merged**